src/components/ModelDialog/constants.ts

```diff
@@ -0,0 +1,4 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+
+export const HEIGHTOFFORMFOOTER = 56
```

Comment:
Still this constant. I thought you'd removed it? Else change the name.

Reply:
I did, must have made a mistake when committing. Removed it again.